@here/harp-mapview/lib/MapView.ts

```diff
@@ -3701,7 +3662,12 @@ export class MapView extends EventDispatcher {
         // no need to check everything if we're not going to create text renderer.
         renderList.forEach(({ zoomLevel, renderedTiles }) => {
             renderedTiles.forEach(tile => {
-                this.renderTileObjects(tile, zoomLevel);
+                this.m_tileObjectRenderer.render(
+                    tile,
+                    zoomLevel,
+                    this.m_camera.position,
+                    this.m_sceneRoot
```

Comment:
I wonder if the scene root node can be owned by the `TileObjectRenderer` ?

It would require moving the `MapAnchors` update method as well to the `TileObjectRenderer`, but that may be actually a good idea?

At least the `update` method has similar logic to the `renderTileObjects` method, i.e. position and add to the root

Reply:
Hmm, I see TileObjectsRenderer more in parallel to the MapAnchors, although the naming does not obviously say so. I could call the new class only TileObjects without the "Renderer" suffix, that might clear things up a bit. what do you think?

Reply:
I valid name is hard to come up, the change is anyway good.